pages/dkp/konvoy/2.2/choose-infrastructure/azure/advanced/new/index.md

```diff
@@ -18,29 +18,24 @@ enterprise: false
 1.  Set the environment variable:
 
     ```bash
-    CLUSTER_NAME=my-azure-cluster
+    export CLUSTER_NAME=azure-example
     ```
 
 ## Tips and Tricks
 
 1.  To create a cluster name that is unique, use the following command:
 
     ```bash
-    CLUSTER_NAME=$(whoami)-azure-cluster-$(LC_CTYPE=C tr -dc 'a-z0-9' </dev/urandom | fold -w 5 | head -n1)
+    CLUSTER_NAME=azure-example-azure-cluster-$(LC_CTYPE=C tr -dc 'a-z0-9' </dev/urandom | fold -w 5 | head -n1)
```

Comment:
Does this step assume `CLUSTER_NAME` was already exported, e.g., in the previous step? It might be a good idea to repeat the `export` here.

Reply:
I see you repeated the export in another section in this PR: https://github.com/mesosphere/dcos-docs-site/pull/4191/files#diff-305dbabc0cdf3f5f85f232d14861b3dd1928f74fc41e2b314cecacc24b044dbcR44
```suggestion
    export CLUSTER_NAME=azure-example-azure-cluster-$(LC_CTYPE=C tr -dc 'a-z0-9' </dev/urandom | fold -w 5 | head -n1)
```

Reply:
@dlipovetsky  Thank you

Reply:
this commit wasn't added :(